lockbox-ios/View/ItemDetailView.swift

```diff
@@ -79,15 +79,25 @@ extension ItemDetailView: ItemDetailViewProtocol {
     }
 
     var rightButtonText: AnyObserver<String?> {
-        // swiftlint:disable:next force_cast
-        return (self.navigationItem.rightBarButtonItem!.customView as! UIButton).rx.title().asObserver()
+        return AnyObserver<String?> { evt in
+            // swiftlint:disable:next force_cast
+            let button = (self.navigationItem.rightBarButtonItem!.customView as! UIButton)
+            button.setTitle(evt.element ?? "", for: UIControl.State.normal)
+            button.sizeToFit()
+
+        }
     }
 
     var leftButtonText: AnyObserver<String?> {
         // swiftlint:disable:next force_cast
         return (self.navigationItem.leftBarButtonItem!.customView as! UIButton).rx.title().asObserver()
     }
 
+    var leftButtonIcon: AnyObserver<UIImage?> {
+        // swiftlint:disable:next force_cast
+        return (self.navigationItem.leftBarButtonItem!.customView as! UIButton).rx.image().asObserver()
```

Comment:
@joeyg I'm not sure how but I think the left button is still appearing during selected/highlighted states.

<img width="385" alt="Screen Shot 2019-06-28 at 2 46 39 PM" src="https://user-images.githubusercontent.com/49511/60371617-07c61e00-99b7-11e9-93da-c4b464e642a3.png">


In other words, if you tap the "Cancel" button and hold it down you'll see the back arrow "<" appear.

I couldn't figure out how I could fix this myself though...

Reply:
Filed follow up at #1082